controllers/feeds/getFeedsV2.js

```diff
@@ -182,7 +192,7 @@ module.exports = async (req, res) => {
             item.is_self =
               item.actor.id === req.userId || item.actor.id === myAnonymousUser?.user_id;
             let newItem = await modifyFeedIsFollowingTarget(item, req.userId);
-            newItem = await modifyAnonimityPost(newItem);
+            newItem = await modifyAnonimityPost(newItem, isBlurredPost);
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

This change introduces a new parameter `isBlurredPost` to the `modifyAnonimityPost` function. It seems like this parameter is used to determine whether the post should be blurred or not. This is a good addition as it allows more flexibility in how posts are displayed based on the number of followers of the post's author. However, please ensure that the `modifyAnonimityPost` function handles the case when `isBlurredPost` is `undefined`, as this could potentially lead to unexpected behavior.


<!-- This is an auto-generated comment by OSS CodeRabbit -->